ke/compiler.py

```diff
@@ -89,6 +89,47 @@
     builtin_macros["#" + short] = builtin_macros["#" + long]
 
 
+def compile_separate(separate, expr):
+    print('\n\n\n\n#########')
+
+    if separate is None:
+        raise CompileError("Must specify a valid separator: ',', ':', '|'")
+    print(f"incoming separator of: '{separate}' - '{expr}'")
+    (a, b, is_greedy, sub) = expr
```

Comment:
communicate somehow that you expect `expr` to be a `Multiple`, e.g. check it and raise a nice error if it isn't